ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -1887,7 +1887,7 @@ public sealed class ArchiWebHandler : IDisposable {
 				return null;
 			}
 
-			index += 20;
+			index += "g_daysTheirEscrow = ".Length;
```

Comment:
If you could move `"g_daysTheirEscrow = "` to `const string` and use it in both places, it'd for sure be better.